Update to EasyCodingStandard 4

TomasVotruba opened the pull request:
Few weeks ago ECS 4 was released:https://www.tomasvotruba.cz/blog/2018/03/26/new-in-easy-coding-standard-4-clean-symfony-standard-with-yaml-and-services/

Here is just casual upgrade :) mostly Neon→Yaml

TomasVotruba commented on src/Classification/NaiveBayes.php:
```diff
@@ -155,7 +155,7 @@ private function sampleProbability(array $sample, int $feature, string $label):
         // scikit-learn did.
         // (See : https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/naive_bayes.py)
-        $pdf = -0.5 * log(2.0 * pi() * $std * $std);
+        $pdf = -0.5 * log(2.0 * M_PI * $std * $std);
```
This was upgraded in PHP CS Fixer. Do you agree with it?

akondas replied:
Yes :+1:

TomasVotruba commented on tests/Tokenization/WordTokenizerTest.php:
```diff
@@ -33,8 +33,8 @@ public function testTokenizationOnUtf8(): void
-                  '剆坲', '煘煓瑐', '鬐鶤鶐', '飹勫嫢', '枲柊氠', '鍎鞚韕', '焲犈',
-                  '殍涾烰', '齞齝囃', '蹅輶', '孻憵', '擙樲橚', '藒襓謥', '岯岪弨', '廞徲', '孻憵懥', '趡趛踠', ];
+            '剆坲', '煘煓瑐', '鬐鶤鶐', '飹勫嫢', '枲柊氠', '鍎鞚韕', '焲犈',
+            '殍涾烰', '齞齝囃', '蹅輶', '孻憵', '擙樲橚', '藒襓謥', '岯岪弨', '廞徲', '孻憵懥', '趡趛踠', ];
```
Again, added in new PHP CS Fixers. Is that ok?

akondas replied:
:+1:

akondas commented:
Any idea why build fails on iOS:
```
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Package operations: 77 installs, 0 updates, 0 removals
  - Installing ocramius/package-versions (1.3.0): Downloading (100%)
Parse error: parse error, expecting `';'' or `'{'' in /Users/travis/build/php-ai/php-ml/vendor/ocramius/package-versions/src/PackageVersions/Installer.php on line 59
```

TomasVotruba commented:
It links here: https://github.com/Ocramius/PackageVersions/blob/1.3.0/src/PackageVersions/Installer.php#L59

Probably PHP 7.0 error

TomasVotruba commented:
What's the blocker here?

akondas commented:
Build on iOs fails, restarted, but I do not have time to look at the case now :(

TomasVotruba commented:
I see, there is new error.
Could you retry build?

akondas commented:
I found the cure #281

akondas commented:
@TomasVotruba rebase will be required and we can merge

TomasVotruba commented:
- :heavy_check_mark: Rebased, 
- :heavy_check_mark: updated to ECS 4.4, with the most recents PHP CS Fixer and PHP_CodeSniffer versions
- :heavy_check_mark: fixed cs
- :heavy_check_mark: tests pass :heavy_check_mark:

akondas commented:
Thanks :+1:

TomasVotruba commented:
Happy to help :+1:
